tests/test_server_auth.py

```diff
@@ -581,6 +581,11 @@ async def test_server_auth_in_transaction(self):
                 DROP ROLE foo;
             ''')
 
+    @unittest.skipIf(
+        platform.system() == "Darwin" and platform.machine() == 'x86_64',
+        "Postgres is not getting getting enough shared memory on macOS "
```

Comment:
> on macOS

Probably on the `macos-14` GitHub runner? This test passed on my local x86 macOS 14.

Reply:
I've updated the message.